.github/workflows/phpunit.yml

```diff
@@ -0,0 +1,66 @@
+name: "PHPUnit tests"
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  phpunit:
+    name: "PHPUnit tests"
+
+    runs-on: ${{ matrix.operating-system }}
+
+    strategy:
+      matrix:
+        dependencies:
+          - "lowest"
+          - "highest"
+          - "locked"
+        php-version:
+          - "7.3"
+          - "7.4"
+          - "8.0"
+        operating-system:
+          - "ubuntu-latest"
+          - "windows-latest"
+          - "macos-latest"
+        include:
+          - php-version: "8.0"
+            dependencies: "highest"
+            operating-system: "ubuntu-latest"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          coverage: "pcov"
+          php-version: "${{ matrix.php-version }}"
+          ini-values: memory_limit=-1
+          tools: composer:v2, cs2pr
+
+      - name: "Cache dependencies"
+        uses: "actions/cache@v2"
+        with:
+          path: |
+            ~/.composer/cache
```

Comment:
Don't think this will cache well on Windows, [something like this should work better](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/e029b62cbb759eb2596184acf44314a04aa400ac/.github/workflows/ci.yml#L91-L104).

Reply:
also here, preferring simplicity and keeping windows as a black sheep for a while.

Will eventually upgrade later